app/src/main/java/com/kickstarter/libs/featureflag/FeatureFlagClient.kt

```diff
@@ -7,10 +7,21 @@ import com.kickstarter.libs.Build
 import com.kickstarter.libs.Build.isInternal
 import com.kickstarter.libs.featureflag.FeatureFlagClient.Companion.INTERNAL_INTERVAL
 import com.kickstarter.libs.featureflag.FeatureFlagClient.Companion.RELEASE_INTERVAL
+import com.kickstarter.models.UserPrivacy
+import io.reactivex.Observable
 import timber.log.Timber
 
 interface FeatureFlagClientType {
 
+    /**
+     * Backend list of features flags enabled within `userPrivacy.enabledFeatures` field
+     *
+     * Checks if the FlipperFlagKey.name is present within enabledFeatures
+     */
+    fun isBackendEnabledFlag(privacy: Observable<UserPrivacy>, key: FlipperFlagKey): Observable<Boolean> {
+        return privacy.map { it.enabledFeatures.contains(key.key) }
```

Comment:
Take a look at the query: 
![Screenshot 2024-09-12 at 9 01 40 AM](https://github.com/user-attachments/assets/83e7f343-db5f-4021-aa3f-0bf441aa0eab)